crates/ruff_linter/src/rules/pyflakes/rules/unused_import.rs

```diff
@@ -104,14 +106,55 @@ impl Violation for UnusedImport {
 
     fn fix_title(&self) -> Option<String> {
         let UnusedImport { name, multiple, .. } = self;
+        let resolution = match self.context {
+            Some(UnusedImportContext::Init {
+                first_party: true,
+                dunder_all: true,
+            }) => "Add unused import to __all__",
+            Some(UnusedImportContext::Init {
+                first_party: true,
+                dunder_all: false,
+            }) => "Use a redundant alias",
+            _ => "Remove unused import",
+        };
         Some(if *multiple {
-            "Remove unused import".to_string()
+            resolution.to_string()
         } else {
-            format!("Remove unused import: `{name}`")
+            format!("{resolution}: `{name}`")
```

Comment:
I guess it's not new, but I'm finding the selective inclusion of `name` here to be confusing as a user, since we show one diagnostic at a time that underlines the import, but then _sometimes_ include the name and sometimes don't. I'd vote to just remove `name` here while we're changing it.

Reply:
(Though this only shows up in the fix message?)

Reply:
Sounds like no change is needed here.